service/internal/access/v2/just_in_time_pdp.go

```diff
@@ -94,10 +100,17 @@ func (p *JustInTimePDP) GetDecision(
 	case *authzV2.EntityIdentifier_Token:
 		entityRepresentations, err = p.resolveEntitiesFromToken(ctx, entityIdentifier.GetToken(), skipEnvironmentEntities)
 
-	// TODO: implement this case
 	case *authzV2.EntityIdentifier_RegisteredResourceValueFqn:
-		p.logger.DebugContext(ctx, "getting decision - resolving registered resource value FQN")
-		return nil, false, errors.New("registered resources not yet implemented")
+		valueFQN := strings.ToLower(entityIdentifier.GetRegisteredResourceValueFqn())
```

Comment:
Maybe we should clarify which types of values we're using when we use `valueFQN` throughout this library (i.e. `attrValueFQN` or `regResValueFQN`).

Reply:
What do you think?

Reply:
Yea, I think that's a good idea. Will refactor at the end.